posthog/temporal/tests/batch_exports/test_postgres_batch_export_workflow.py

```diff
@@ -659,7 +663,7 @@ async def test_postgres_export_workflow_without_events(
     "data_interval_start",
     # This is hardcoded relative to the `data_interval_end` used in all or most tests, since that's also
     # passed to `generate_test_data` to determine the timestamp for the generated data.
-    [dt.datetime(2023, 4, 24, 15, 0, 0, tzinfo=dt.UTC)],
+    [dt.datetime.now(tz=dt.UTC).replace(hour=0, minute=0, second=0, microsecond=0) - dt.timedelta(hours=24)],
     indirect=True,
 )
 @pytest.mark.parametrize("interval", ["hour"], indirect=True)
```

Comment:
Shouldn't this be `- dt.timedelta(hours=1)` given that we are only testing hourly?

Reply:
from my understanding, this test is testing when data exists outside the data interval (see below on line 683):
> We expect persons outside the batch interval to also be backfilled (i.e. persons that were updated more than an hour ago) when setting `is_earliest_backfill=True`.

Also the data interval in this test was previously 24 hours: `data_interval_start` was  `dt.datetime(2023, 4, 24, 15, 0, 0, tzinfo=dt.UTC)` and `data_interval_end` was by default `dt.datetime(2023, 4, 25, 15, 0, 0, tzinfo=dt.UTC`